tests/e2e-tests/specs/front-end/front-end-cart.test.js

```diff
@@ -8,11 +8,17 @@
 import { createSimpleProduct } from '../../utils/components';
 import { CustomerFlow, StoreOwnerFlow } from '../../utils/flows';
 import { uiUnblocked } from '../../utils';
+import { factories } from '@woocommerce/e2e-factories'
 
 describe( 'Cart page', () => {
 	beforeAll( async () => {
 		await StoreOwnerFlow.login();
-		await createSimpleProduct();
+
+		await factories.product.create( {
+			Name: 'Simple Product',
+			RegularPrice: '9.99',
+		} );
+
```

Comment:
depends on your setup, but it's much preferable to run a fixture import in the global setup phase so that you only import once

Reply:
I usually prefer to keep test data generation as close to the tests as possible in order to allow for parallelization. That said, this was just intended to be an example usage of the package, as opposed to a solid decision about how we're going to consume it in Core.